pagopa/io-dev-api-server
# chore: Show error numbers in `login.html`

## Short description
This PR shows the error number inside the dropdown list item description too.

## How to test
Run the app against the dev server and check the error dropdown list.

<details /open><summary>Details</summary>
<p>

<img src="https://github.com/pagopa/io-dev-api-server/assets/16268789/11be44e2-ef4e-42ee-a8f2-5bc513956f7f" width="250" />

</p>
</details>
